frontend/src/app/models/user/user-dto.ts

```diff
@@ -2,4 +2,5 @@ export interface UserDto {
     id : number;
     email : string;
     workspaceName : string;
+    avatarLink : string;
```

Comment:
Also find one formatting time, somewhere it is
avatarLink:string
Somewhere
avatarLink : string